crates/testing/tests/tests_1/libp2p.rs

```diff
@@ -44,6 +44,7 @@ async fn libp2p_network() {
 /// libp2p network test with failures
 #[cfg_attr(async_executor_impl = "tokio", tokio::test(flavor = "multi_thread"))]
 #[cfg_attr(async_executor_impl = "async-std", async_std::test)]
+#[ignore]
```

Comment:
Is this test not working?

Reply:
Just noisy, so I ignore it until I get the non failure one going.